lib/features/game/views/page_view_pages/count_down_page.dart

```diff
@@ -8,10 +8,8 @@ class CountDownPage extends StatelessWidget {
 
   List<Widget> getBody(Orientation orientation, int questionNumber) {
     return [
-      if (orientation == Orientation.portrait) SizedBox(height: 60),
-      HeaderText(text: 'WAIT FOR ' + questionNumber.toString() + ' QUESTION'),
-      SizedBox(height: 60),
-      CircularCountdown(time: 3),
```

Comment:
Magic number make const out of it

Reply:
done

Reply:
How is that const... and also save it in servers consts